frontend/src/components/modals/MultiplePluginsInstallModal.tsx

```diff
@@ -80,7 +80,10 @@ const MultiplePluginsInstallModal: FC<MultiplePluginsInstallModalProps> = ({
       onOK={async () => {
         setLoading(true);
         await onOK();
-        setTimeout(() => Navigation.OpenQuickAccessMenu(QuickAccessTab.Decky), 250);
+        setTimeout(() => {
+          Navigation.OpenQuickAccessMenu(QuickAccessTab.Decky);
+          DeckyPluginLoader.setDesktopMenuOpen(true);
```

Comment:
Similar to above comment, we *could*  make `OpenQuickAccessMenu` open the desktop menu but eeh i don't know if any plugins use it, and it only makes sense for the decky tab so not really sure.